packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
                 v-model="activePage"
-                :visible-pages="4"
                 :pages="15"
+                :visible-pages="4"
                 color="warning"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Disabled Pagination</td>
+          <td>Disabled</td>
           <td>
             <VbCard>
               <va-pagination
                 v-model="activePage"
-                :visible-pages="4"
                 :pages="15"
+                :visible-pages="4"
                 disabled
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Pagination Without Links</td>
+          <td>Without Links</td>
           <td>
             <VbCard>
               <va-pagination
+                v-model="activePage"
+                :boundary-links="false"
                 :pages="10"
                 :visible-pages="3"
-                :boundary-links="false"
-                v-model="activePage"
               />
               <va-pagination
+                v-model="activePage"
+                :direction-links="false"
                 :pages="10"
                 :visible-pages="3"
-                :direction-links="false"
-                v-model="activePage"
               />
             </VbCard>
           </td>
         </tr>
         <tr>
-          <td>Pagination With Icons</td>
+          <td>Icons</td>
           <td>
             <VbCard>
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                direction-icon-left='volume_mute'
-                direction-icon-right='volume_down'
-                v-model="activePage"
+                direction-icon-left="volume_mute"
+                direction-icon-right="volume_down"
               />
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                boundary-icon-left='volume_off'
-                boundary-icon-right='volume_up'
-                v-model="activePage"
+                boundary-icon-left="volume_off"
+                boundary-icon-right="volume_up"
               />
               <va-pagination
+                v-model="activePage"
                 :pages="10"
                 :visible-pages="3"
-                boundary-icon-left='volume_off'
-                boundary-icon-right='volume_up'
-                direction-icon-right='volume_down'
-                direction-icon-left='volume_mute'
+                boundary-icon-left="volume_off"
+                boundary-icon-right="volume_up"
+                direction-icon-left="volume_mute"
+                direction-icon-right="volume_down"
+              />
+            </VbCard>
+          </td>
+        </tr>
+        <tr>
+          <td>Input</td>
+          <td>
+            <VbCard>
+              <va-pagination
```

Comment:
Here would be nice to have an ability to remove navigation arrows, so that it looks like this

<img width="164" alt="image" src="https://user-images.githubusercontent.com/5394573/189234764-e174ecf7-35d4-436a-bdca-03904b270892.png">

and this

<img width="249" alt="image" src="https://user-images.githubusercontent.com/5394573/189234784-f9082b3d-96d2-4d93-a799-c8b39f548437.png">